src/main/java/com/ea/restaurant/util/GrpcUtil.java

```diff
@@ -0,0 +1,14 @@
+package com.ea.restaurant.util;
+
+import java.nio.charset.StandardCharsets;
+import java.util.Base64;
+
+public class GrpcUtil {
+
+  public static String[] getCredentialsFromBasicAuthToken(String authorizationValue) {
```

Comment:
Don't use arrays, use List instead

Reply:
Send the metadata to this function instead of one value of the authorization.

Reply:
Done.